### PR TITLE
Disable tokens endpoints

### LINKED ISSUE #97: Feature to disable tokens endpoints
Following the feature
- https://github.com/kaprien/kaprien-rest-api/issues/96

The possibility of disabling the endpoint for tokens could be interesting. That way, users that deploy multiple Rest APIs could choose to disable the endpoint for nodes that are outside the managing network, for example.

### PR DESCRIPTION
**Pull request by juniorlpa**
Implementing the feature to disable the endpoint for tokens.

This feature gives the ability to users disable the endpoint for nodes on cases where there are multiple Rest APIs outside of the managing network.

Variable: `RSTUF_TOKENS_NODE`
Default : `true`

Closes #97

### CODE REVIEW

File: docs/source/guide/Docker_README.md [hunk 1]
```diff
@@ -38,6 +38,15 @@ docker run -p 80:80 \
 
 ### Environment Variables
 
+#### (Optional) `RSTUF_TOKENS_NODE`
+
+The value type is boolean (true/false [case sensitive](https://www.dynaconf.com/configuration/#available-options)).
+Default: true
+
+Disble the container as a token node.
```
Comment (rdimitrov) [inline docs/source/guide/Docker_README.md:46]: s/disble/disable

**Review by rdimitrov — COMMENTED** [review]
Hey, I don't seem to see any obvious changes, but was there a reason why we update the figures?

**Comment by kairoaraujo** [comment]
> Hey, I don't seem to see any obvious changes, but was there a reason why we update the figures? 

Still not optimal documentation build. 
It rebuilds the images from the platinum

**Comment by MVrachev** [comment]
I will review it later today.

**Review by MVrachev — COMMENTED** [review]
Thanks for the changes @juniorlpa!

Can I ask you to reformat the commit title and message of your second commit: https://github.com/vmware/repository-service-tuf-api/pull/161/commits/d162284841193edf6ad07ec80ad29ac5203bc8ea?

There are some general guidelines around git commits and we do our best to follow them: 
https://cbea.ms/git-commit/

**Comment by kairoaraujo** [comment]
I also think you can remove the images updates (they are not relevant for this change)
```git checkout main -- docs/source/_static```

**Comment by kairoaraujo** [comment]
Hi @juniorlpa, I updated the minor changes to @MVrachev @rdimitrov review it.
I think it is a great PR for our v0.0.1a3

**Review by rdimitrov — APPROVED** [review]
Awesome! 🚀

**Pull request merged**